tests-behat/bootstrap/Context.php

```diff
@@ -542,6 +569,23 @@ public function compareInputValueToElementText($inputName, $selector)
         }
     }
 
+    /**
+     * Find a dropdown component within an html element
+     * and check if value is set in dropdown.
+     */
+    private function assertDropdownValue(NodeElement $element, string $value, string $selector)
+    {
+        $dropdown = $element->find('css', $selector);
+        if (!$dropdown) {
+            throw new \Exception('Dropdown input not found using selector: ' . $selector);
```

Comment:
Even I normally do not like to import non namespaced classes, I would recommend to import/add `use Exception` to stress that we want to throw this type of exceptions in Behat Context.

Reply:
done!